### PR TITLE
test: add a test for `exchange_utils`

### PR DESCRIPTION
**Pull request by delaaxe**
# Pull Request type

Please add the labels corresponding to the type of changes your PR introduces:

- Testing

## What is the current behavior?

Resolves: #281

## Other information

blocked by this foundry issue: https://github.com/foundry-rs/starknet-foundry/issues/659

### CODE REVIEW

**Comment by delaaxe** [comment]
blocked by this foundry issue: https://github.com/foundry-rs/starknet-foundry/issues/659

**Comment by delaaxe** [comment]
@zarboq @sparqet you can review

File: tests/exchange/test_exchange_utils.cairo [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+use starknet::info::get_block_number;
+use snforge_std::{declare, start_prank, stop_prank, start_roll, ContractClassTrait};
+
+use satoru::data::keys;
+use satoru::data::data_store::{IDataStoreDispatcher, IDataStoreDispatcherTrait};
+use satoru::exchange::exchange_utils::validate_request_cancellation;
+use satoru::tests_lib::{setup, teardown};
+
+#[test]
+fn test_exchange_utils() {
```
Comment (zarboq) [inline tests/exchange/test_exchange_utils.cairo:14]: rename test to follow our naming conventions

ex: `given_division_by_0_when_roundup_magnitude_division_then_fails`

**Review by zarboq — CHANGES_REQUESTED** [review]
Test naming issue, else everything looks good :fire:

**Review by zarboq — APPROVED** [review]

**Pull request merged**